modules/local/add_summary_to_metadata.nf

```diff
@@ -2,9 +2,6 @@ process ADD_SUMMARY_TO_METADATA {
     tag "$meta.id"
     label 'process_single'
 
-    if (params.enable_conda) {
-        exit 1, "Conda environments cannot be used when using the ADD_SUMMARY_TO_METADATA module. Please use docker or singularity containers."
-    }
```

Comment:
The convention is to do something like this: https://github.com/nf-core/modules/blob/master/modules/nf-core/fastk/fastk/main.nf#L8-L11